algebras/algebra/src/main/scala/endpoints/algebra/StatusCodes.scala

```diff
@@ -0,0 +1,21 @@
+package endpoints.algebra
+
+/**
+  * @group algebras
+  */
+trait StatusCodes {
+
+  /** HTTP Status Code */
+  type StatusCode
+
+  /** 2xx Success */
+  def OK: StatusCode
+
+  /** 4xx Client Error */
+  def Unauthorized: StatusCode
+
+  def NotFound: StatusCode
+
+  def BadRequest: StatusCode
+
+}
```

Comment:
Do we want to add `statusCodeFromInt(value: Int): StatusCode` convenience method? (I’m not sure it will be implementable in all interpreters, though)

Reply:
Could be added latter if we see it would much easier to add a code with it, but honestly most of the time it's either already an Int, or it's the exact same same as the algebra.